ci/tools.go

```diff
@@ -1,3 +1,4 @@
+//go:build tools
```

Comment:
not sure but should this be here?

Reply:
I think so. This has to do with how build directives are specified in later versions of go: https://pkg.go.dev/cmd/go#hdr-Build_constraints

https://go.googlesource.com/proposal/+/master/design/draft-gobuild.md#transition talks about the proposed transition. I could probably remove the old directive in fact

Reply:
There is a tool we can run later to clean up all references in the old format. Not a problem and the old directives dont break things just are non functional

Reply:
just for [ref](https://pkg.go.dev/cmd/go#hdr-Build_constraints)
Go versions 1.16 and earlier used a different syntax for build constraints, with a "// +build" prefix. The gofmt command will add an equivalent //go:build constraint when encountering the older syntax.